code/BuddyTrackerNorth/BuddyTrackerNorth.ino

```diff
@@ -70,11 +70,11 @@ void loop() {
 }
 
 int orientation2LED(float orientation){
-    int fov = 180;
```

Comment:
I would use float even until you find it too slow. Truncation and rounding can cause problems, so you truncate or round at the very end if possible. Also a good idea to check in a unit test that this method always returns expected results.